server/market/market_test.go

```diff
@@ -859,7 +860,6 @@ func TestMarket_Run(t *testing.T) {
 	<-auth.handlePreimageDone
 	// and for matching to complete (in processReadyEpoch).
 	<-storage.epochInserted
-	storage.epochInserted = nil
```

Comment:
That's the last epoch with this test market anyway, so there are no more sends from TArchivist on this channel anyway.  Remove this since it was unguarded and technically a race.